### DISCUSSION TITLE
Can't open because I deleted the Fortnite path. How to fix?

### DISCUSSION THREAD

**Discussion by HamSpamMcbam** [4sval/FModel#460] [Q&A]
[FModel-Log-2024-03-08.txt](https://github.com/4sval/FModel/files/14544751/FModel-Log-2024-03-08.txt)

Tried using this for another game and got nothing but Fortnite results so I deleted the Fortnite path, resulting in this error message: "An unhandled exception occurred: The invocation of the constructor on type 'FModel.MainWindow' that matches the specified binding constraints threw an exception."
The FAQ/installation says to look in this log to find out why it's crashing. Only issue is, I can't fix the issue without opening the program but the program won't open because of this issue. Is there a way to put the path back without opening the program somehow?

**Comment by 4sval** [comment]
please uninstall games properly next time...
as for the fix, either recreate the folder you deleted or delete `%appdata%/fmodel/`
